packages/flame_3d/lib/src/parser/gltf/image.dart

```diff
@@ -0,0 +1,76 @@
+import 'dart:typed_data';
+import 'dart:ui' as dart;
+
+import 'package:flame_3d/resources.dart' as flame3d;
+import 'package:flame_3d/src/parser/gltf/buffer_view.dart';
+import 'package:flame_3d/src/parser/gltf/gltf_node.dart';
+import 'package:flame_3d/src/parser/gltf/gltf_node_with_data.dart';
+import 'package:flame_3d/src/parser/gltf/gltf_ref.dart';
+import 'package:flame_3d/src/parser/gltf/gltf_root.dart';
+import 'package:flame_3d/src/parser/gltf/mime_type.dart';
+
+/// Image data used to create a texture.
+class Image extends GltfNode with GltfNodeWithData<flame3d.ImageTexture> {
```

Comment:
I think it would be good to make this something else, since it'll clash with default flutter and dart imports and people will import the wrong class and be confused.

Reply:
Or is it only instantiated by us? Then it doesn't matter

Reply:
this is all only internally used and matches the name on the gltf spec